pages/tutorials/consent-management/consent-management-configuration/_meta.json

```
@@ -6,5 +6,6 @@
   "google-tag-manager": "Configuring Google Tag Manager",
   "global-privacy-control": "Configuring Global Privacy Control (GPC)",
   "css-styling": "Customizing Appearance with CSS",
-  "privacy-notices-localization": "Optional: Configuring Custom Notices for Localization"
+  "privacy-notices-localization": "Optional: Configuring Custom Notices for Localization",
+  "consent-monitoring": "Optional: Monitoring Consent"
```

```suggestion
  "consent-reporting": "Consent Reporting",
  "privacy-notices-localization": "Optional: Configuring Custom Notices for Localization"
```

I'd recommend we call this "Consent Reporting" (and rename the page `consent-reporting.mdx` accordingly). It's more discoverable that way